cloud/aws/bin/deploy.py

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env python3
+import subprocess
+import os
+import sys
+
+# TODO(#2743): move this to deploy specific script.
+sys.path.append(os.getcwd())
+
+from cloud.shared.bin.lib import civiform_mode
+from cloud.shared.bin.lib import terraform
+
+# Keep in sync with cloud/shared/bin/lib.sh
+os.environ['TF_VAR_FILENAME'] = 'setup.auto.tfvars'
```

Comment:
On second glance, we might be able to do away with the environment variable setting entirely if downstream scripts are written in Python. The reason being that Python scripts could just encode these in helper functions / packages.

I think we're likely not there yet, but just noting that we can hopefully get rid of these in the future.